apps/stats/src/views/Stats/Growth/Growth.tsx

```
@@ -0,0 +1,241 @@
+            return `Which posts drove the most growth ${getPeriodText(range)}`;
+        }
+    };
+
+    const isPageLoading = isLoading;
+
+    return (
+        <StatsLayout>
+            <StatsHeader>
+                <DateRangeSelect />
+            </StatsHeader>
+            <StatsView data={isPageLoading ? undefined : chartData} isLoading={false} loadingComponent={<></>}>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)